waltz-ng/client/data-types/components/data-type-tree.html

```diff
@@ -0,0 +1,53 @@
+<waltz-section name="Data Types" small="All">
+
+    <ul class="list-unstyled"
+        ng-show="$ctrl.dataTypeHierarchy.length > 0">
+        <li ng-repeat="dataType in $ctrl.dataTypeHierarchy | orderBy:'name'">
+            <div ng-include="'node.html'"
+                 ng-init="node=dataType; level=1">
+            </div>
+        </li>
+    </ul>
+
+    <div class="alert alert-warning"
+         ng-show="$ctrl.dataTypeHierarchy.length == 0">
+        <strong>No Data Types</strong> have been defined within this system.
+    </div>
+
+</waltz-section>
+
+
+<script type="text/ng-template"
+        id="node.html">
```

Comment:
Lifted this from the org-unit list-view.html where it also has the name node.html.  They don't appear to require a unique name and interfere with one another but will give them both unique names.